app/scripts/modules/azure/serverGroup/configure/wizard/basicSettings/basicSettings.html

```diff
@@ -51,8 +51,8 @@
         <div class="col-md-9">
           <ui-select class="form-control input-sm" required ng-model="command.imageName" on-select="basicSettingsCtrl.imageChanged($item)">
             <ui-select-match placeholder="Pick an image">{{$select.selected.imageName}}</ui-select-match>
-            <ui-select-choices repeat="image.imageName as image in command.images | anyFieldFilter: {imageName: $select.search, ami: $select.search } | orderBy: '-imageName'">
-              <span ng-bind-html="image.imageName | highlight: $select.search"></span>
+            <ui-select-choices repeat="image.imageName as image in command.images | regional:command.region ">
```

Comment:
Sorry, I meant the filtering that happens when the user types into the search field, e.g.
<img width="464" alt="screen shot 2016-03-08 at 9 00 08 am" src="https://cloud.githubusercontent.com/assets/73450/13609039/3ce50096-e50c-11e5-930a-86c8902d787f.png">
I think you just need another filter after the `regional` filter:
```html
<ui-select-choices repeat="image.imageName as image in command.images | regional:command.region | filter:$select.search">
```
Alternatively, if it's a small list, you can omit the search functionality entirely from the select box by setting `search-enabled: false` on the `<ui-select>` tag.